test/parallel/test-fs-rm.js

```diff
@@ -280,3 +282,44 @@ function removeAsync(dir) {
     message: /^The value of "options\.maxRetries" is out of range\./
   });
 }
+
+{
+  if (!common.isIBMi && (common.isWindows || process.getuid() !== 0)) {
+    function makeDirectoryReadOnly(dir) {
+      let accessErrorCode = 'EACCES';
+      if (common.isWindows) {
+        accessErrorCode = 'EPERM';
+        execSync(`icacls ${dir} /inheritance:r`);
+        execSync(`icacls ${dir} /deny "everyone":W`);
+      } else {
+        fs.chmodSync(dir, '444');
+      }
+      return accessErrorCode;
+    }
+
+    function makeDirectoryWritable(dir) {
+      if (common.isWindows) {
+        execSync(`icacls ${dir} /grant "everyone":W`);
```

Comment:
```suggestion
        execSync(`icacls ${dir} /remove:d "everyone"`);
```

Reply:
This worked locally, so hopefully this will also work in CI